deploy-review/solutions/microservice-schedule/src/main/java/com/redhat/training/conference/ScheduleResource.java

```diff
@@ -0,0 +1,128 @@
+package com.redhat.training.conference;
+
+import java.net.URI;
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.time.LocalTime;
+import java.util.List;
+import java.util.stream.Collectors;
+
+import javax.enterprise.context.RequestScoped;
+import javax.transaction.Transactional;
+import javax.ws.rs.BadRequestException;
+import javax.ws.rs.Consumes;
+import javax.ws.rs.DELETE;
+import javax.ws.rs.GET;
+import javax.ws.rs.NotFoundException;
+import javax.ws.rs.POST;
+import javax.ws.rs.PUT;
+import javax.ws.rs.Path;
+import javax.ws.rs.PathParam;
+import javax.ws.rs.Produces;
+import javax.ws.rs.core.GenericEntity;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+
+@RequestScoped
+@Path("/schedule")
+@Produces(MediaType.APPLICATION_JSON)
+public class ScheduleResource {
```

Comment:
To me, in any of our exercise codes, all endpoints must include openapi specs and the correct HTTP codes.